figure_notebooks/supp_fig1.md

```diff
@@ -243,10 +243,17 @@ ggsave(file.path(proj_dir, "/figure_notebooks/tmp_files/supp_fig1_tmp/gg_loc_lin
 ``` r
 gg_gender = image_read_pdf(file.path(proj_dir,
                                   "/figure_notebooks/tmp_files/supp_fig1_tmp/gg_gender.pdf"))
+gg_gender = image_annotate(gg_gender, "a", size = 30)
+
+
 gg_loc_hist = image_read_pdf(file.path(proj_dir,
                                   "/figure_notebooks/tmp_files/supp_fig1_tmp/gg_loc_hist.pdf"))
+gg_loc_hist = image_annotate(gg_loc_hist, "b", size = 30)
+
+
 gg_loc_line = image_read_pdf(file.path(proj_dir,
                                   "/figure_notebooks/tmp_files/supp_fig1_tmp/gg_loc_line.pdf"))
+gg_loc_line = image_annotate(gg_loc_line, "c", size = 30)
```

Comment:
Nice figures for the supplement. Is there anyway to fix the label overlap on the C figure? If not then no worries.